tests/test_pandas.py

```diff
@@ -13,6 +13,7 @@
 import pandas as pd
 from pandas import Grouper
 from pandas.api.extensions import ExtensionArray
+import pandas.util as pdutil
```

Comment:
Maybe add a `# TODO: github.com/pandas-dev/pandas/issues/55023` I assume this would fix the workaround?

Reply:
I think so.  In next commit.